packages/payment-integration-api/src/payment/is-credit-card-like.ts

```diff
@@ -0,0 +1,15 @@
+import isVaultedInstrument from './is-vaulted-instrument';
+import { CreditCardInstrument } from './payment';
+
+export default function isCreditCardLike(instrument: unknown): instrument is CreditCardInstrument {
```

Comment:
Perhaps we can rename it to `isCreditCardInstrument`.
It might confuse devs that if there is a `isCreditCardLike`, then what's the difference between `isCreditCardLike` and `isCreditCard`?

Reply:
Done